src/testcafe-runner.js

```diff
@@ -190,7 +192,7 @@ async function run (runCfgPath, suiteName, timeoutSec) {
   }
 
   const region = cfg.runCfg.sauce.region || 'us-west-1';
-  await runReporter({ assetsPath: cfg.assetsPath, region, metadata: cfg.metadata, ...testCafeResults });
+  await runReporter({ assetsPath: cfg.assetsPath, region, metadata: cfg.metadata, saucectlVersion: cfg.saucectlVersion, ...testCafeResults });
```

Comment:
Shouldn't this be that ? 

```suggestion
  await runReporter({ assetsPath: cfg.assetsPath, region, metadata: cfg.metadata, saucectlVersion: cfg.runCfg.saucectlVersion, ...testCafeResults });
```

(Considering `L192` as a reference)

Reply:
I tested it locally, it should be as in my commit.